sql/core/src/test/resources/sql-tests/analyzer-results/change-column.sql.out

```diff
@@ -206,10 +206,10 @@ ALTER TABLE temp_view CHANGE a TYPE INT
 -- !query analysis
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "UNSUPPORTED_TEMP_VIEW_OPERATION.WITH_SUGGESTION",
+  "errorClass" : "UNSUPPORTED_VIEW_OPERATION.WITHOUT_SUGGESTION",
```

Comment:
to double-check if we are symmetrical: what happens if we use ALTER VIEW to alter a table?

Reply:
Let me to check it.

Reply:
<img width="663" alt="image" src="https://github.com/apache/spark/assets/15246973/5761e465-8161-4e5d-872b-cdf44e6cc0fc">
<img width="639" alt="image" src="https://github.com/apache/spark/assets/15246973/3ec76158-3aba-438b-a57e-7387c57a6e2c">

Reply:
From the test results, it is reasonable and symmetrical.